apps/app.go

```diff
@@ -167,10 +171,12 @@ var DefaultBindingsCall = &Call{
 	Path: DefaultBindingsCallPath,
 }
 
+// App describes an App installed on a Mattermost instance. App should be
+// abbreviated as `app`.
 type App struct {
```

Comment:
A separate PR? I agree that we need a `install --url <xyz> --type http --root-url <abc>` that would override/augment the otherwise "awslambda" manifest.